src/backend/distributed/commands/utility_hook.c

```diff
@@ -644,6 +666,179 @@ ProcessUtilityInternal(PlannedStmt *pstmt,
 		 */
 		CitusHasBeenLoaded();
 	}
+
+	if (list_length(ddlJobs) != 0)
```

Comment:
can we move this logic into a function and maybe call it `ExecutedOnCitusTable `

Reply:
Or, DDLOnCitusTable

Reply:
makes sense, I will address this

Reply:
not relying on ddljobs now, so resolving